Assets/Scripts/Boat/BoatCapacity.cs

```diff
@@ -68,54 +126,71 @@ public int ResetCapacity()
     public void DealDamageToBoat(int damageToTake)
     {
         // If we have no souls on the ferry, and the game is still running, we must be Returning.
-        if (currentLoad == 0)
+        if (CurrentLoad == 0)
         {
             DecreaseCapacity(); 
         }
         // Otherwise, we must be Ferrying.
         else
         {
-            DecreaseSouls(damageToTake);
+            //Add the Souls Lost to the SoulsDamned statistic
+            SoulsDamned = DecreaseSouls(damageToTake);
+            
             // If we reduce the number of souls to zero by taking damage, end the game.
-            if (currentLoad == 0) { OnAllSoulsLost?.Invoke(); }
+            if (CurrentLoad == 0) { OnAllSoulsLost?.Invoke(); }
 
             if (doesLoseCapacityWhileContainsSouls)
             {
                 DecreaseCapacity(damageToTake);
             }
         }
-        if (currentCapacity == 0) { OnBoatDestroyed?.Invoke(); }
+        if (CurrentCapacity == 0) { OnBoatDestroyed?.Invoke(); }
     }
 
     /// <summary>
     /// Add a number of units to the load of the boat. Excess units that couldn't be loaded are returned.
     /// </summary>
     /// <param name="loadToAdd">How many units are we trying to add to the load.</param>
     /// <returns>The number of units that were not able to fit onto the boat due to load capacity limits.</returns>
-    public int IncreaseSouls(int loadToAdd)
+    public int IncreaseSouls(int loadToAdd=int.MaxValue)
     {
-        currentLoad += loadToAdd;
+        CurrentLoad += loadToAdd;
         return ReduceLoadToFitCapacity();
     }
 
+    //The set up event actions requires no return type and no parameters. This function just fulfills that requirement.
+    //TODO replace with Delegates might work?
```

Comment:
You could also do `GameStateManager.OnFerryingEnter += () => IncreaseSouls()`, but this is fine.